packages/next/src/lib/typescript/writeConfigurationDefaults.ts

```diff
@@ -69,6 +69,7 @@ function getDesiredCompilerOptions(
         (ts.ModuleResolutionKind as any).Node12,
         ts.ModuleResolutionKind.Node16,
         ts.ModuleResolutionKind.NodeNext,
+        ts.ModuleResolutionKind.Bundler,
```

Comment:
do we need `as any` for 4.x ts versions?

Reply:
I believe the type here is for our installed version of TS, not for the user's installed version.

So thats why `(Node12 as any)` is necessary because newer TS doesn't have it anymore (I think it was beta only)